config/locales/member_facing.fr.yml

```diff
@@ -1,179 +1,200 @@
+    unknown_error: "Notre équipe technique a été notifiée de ce problème. Veuillez revérifier vos informations ou choisir une autre méthode de paiement."
+  member_registration:
+    check_email: "<strong>Merci pour votre inscription.</strong> <span>Pour activer votre compte, cliquez sur le lien de confirmation qui vient de vous être envoyé par email.</span>"
+    create_account: "Nous traitons votre don."
+    create_account_description: "Maintenant, créez un compte pour gérer vos dons en toute sécurité, ou annuler un don à tout moment."
+    enter_password_message: "Entrez un mot de passe pour commencer"
     form:
-      phone_number: "Mon numéro de téléphone"
-      phone_country_code: "Code"
-      country: "Pays"
-      submit: "Appelez-moi"
-  share:
-    cta: Partagez la campagne avec vos ami(e)s afin de tripler votre impact!
-    send_email: Envoyer un courriel
-    share: Partagez
-    tweet: Tweetez
-    twitter_handle: '@sumofus_fr'
-  errors:
-    probably_invalid: semble invalide
-    is_invalid: est invalide
-    this_field: Ce champ
-    this_field_with_message: Ce champ %{message}
-  validation:
-    is_required: est requis
-    is_invalid_email: n’est pas une adresse email valide
-    is_invalid_phone: 'peut uniquement comprendre des chiffres, tirets, plus, et des parenthèses'
-    is_invalid_country: doit comprendre un code pays valide
-    is_invalid_length: 'doit contenir moins de %{length} caractères'
-    postal:
-      has_invalid_characters: 'Peut uniquement comprendre des lettres, chiffres, espaces et tirets.'
-      too_long: 'Ne peut contenir plus de 9 lettres, chiffres ou caractères.'
-  time:
-    formats:
-      default: "%-d %B %Y"
+      confirm_password: "Confirmez votre mot de passe"
+      password: "Mot de passe"
+      register: "S’inscrire "
+      your_email: "Votre adresse courriel"
+    thanks: "Merci ! Encore une toute dernière étape :"
+    title: Inscription
+  page:
+    more_info: "Plus d’informations"
+    no_script: "Merci de votre intérêt pour ce problème ! Malheureusement, vous ne pouvez pas réaliser cette action tant que Javascript n’est pas activé sur votre navigateur. Merci d’activer Javascript ou d’utiliser un navigateur avec Javascript actif pour pouvoir réaliser cette action."
+  petition:
+    confirmation: "Nom ajouté"
+    excited_confirmation: "Nom ajouté !"
+    fine_print: "Nous protégerons vos données personnelles, et vous tiendrons au courant de la suite ainsi que de nos différentes campagnes."
+    sign_it: "Signez la pétition"
+    target_prefix: "Pétition adressée à"  # FIXME this can be problematic in some cases, unless we manage plural & gender phrases with some kind of magic... I’d recommend leaving user writing the proper article (there’re like five forms in French: À Monsanto, À LA Banque de France, À L’Organisation Mondiale du Commerce, AU Maire de Bordeaux, AUX député-e-s du Parlement Européen).
```

Comment:
From my point of view (and what we’ve ended up doing in our previous tool), I would go for a customizable _target_prefix_ field with either an empty default value, either a default value of “To” in English, “À” in French, “A” in Spanish, etc.

Databases usually provide _articles_ and _hinges_ together with names. It makes sense for hundreds of names (such as cities names), but for a couple of targets... it’s really not an issue to require campaigners to provide those phrases.